### PR TITLE
2.x to become the new master

### PR DESCRIPTION
**Pull request by Nyholm**
This is a PR to show that we have a 2.x branch. 

It should be merged after we release 1.9.

### CODE REVIEW

**Comment by dbu** [comment]
agreed. i'll try to solve the merge conflicts.

File: composer.json [hunk 1]
```diff
@@ -31,12 +36,18 @@
...
+        }
+    },
     "scripts": {
         "test": "vendor/bin/phpspec run",
         "test-ci": "vendor/bin/phpspec run -c phpspec.ci.yml"
     },
     "extra": {
         "branch-alias": {
+            "dev-2.x": "2.x-dev",
             "dev-master": "1.9.x-dev"
```
Comment (dbu) [inline composer.json]: we need to change this

**Comment by Nyholm** [comment]
Thank you for fixing the merge conflicts. 
Im happy with merging now.

**Comment by GrahamCampbell** [comment]
What about dropping PHP 7.0?

**Comment by Nyholm** [comment]
I just suggest to merge to master. Not to tag 2.0. 

About PHP 7.0: https://github.com/php-http/client-common/issues/148

**Pull request merged**